src/preprocessing/passes/symmetry_detect.h

```diff
@@ -65,10 +66,18 @@ class Partition
    * { w-> { x, y } }
    */
   std::map<Node, std::vector<Node> > d_subvar_to_vars;
+  /** add variable */
+  void addVariable(Node sv, Node v);
+  /** remove variable */
+  void removeVariable(Node sv);
```

Comment:
it seems that `removeVariable` is not the "inverse" of `addVariable`. Clarify behavior